worlds/animal_well/names.py

```diff
@@ -463,3 +464,120 @@ def __str__(self) -> str:
     k_medal = "Assembled the K. Medal"
     kangaroo_first_spot = "Kangaroo First Spot"  # first spot the kangaroo appears
     victory_first = "First Victory"
+
+    # fruits, by row and column, top left room is R1C1
+    fruit_0 = "Big Blue Fruit Before Disc Spike Bunny - R1C1a"  # big blue fruit, disc spike room
+    fruit_1 = "Big Blue Fruit After Disc Spike Bunny - R1C1b"  # big blue fruit, disc spike room
+    fruit_2 = "Pink Fruit by Brown Egg Chest - R1C3"
+    fruit_3 = "Blue Fruit Above Dog Fast Travel - R1C4"  # blue fruit
+    fruit_4 = "Blue Fruit by Raw Egg Puzzle Entrance - R1C6a"  # blue fruit
+    fruit_5 = "Pink Fruit Below Raw Egg Puzzle Entrance - R1C6b"
+    fruit_6 = "Pink Fruit Left of Mock Disc Shrine - R1C7"
+    fruit_7 = "Blue Fruit Top of Dog Elevator - R1C9"  # blue fruit
+    fruit_8 = "Pink Fruit in Bear Upper Phone Room Left - R1C11a"
+    fruit_9 = "Pink Fruit in Bear Upper Phone Room Right - R1C11b"
+    fruit_10 = "Pink Fruit Right of Ferrets - R1C15"
+    fruit_11 = "Pink Fruit by Swordfish Lake - R2C3"
+    fruit_12 = "Pink Fruit by Dog Fast Travel - R2C4"
+    fruit_13 = "Blue Fruit by P. Flame - R2C9"  # blue fruit
+    fruit_14 = "Pink Fruit in Dog Elevator by P. Flame - R2C10"
+    fruit_15 = "Blue Fruit by V. Flame - R2C11"  # blue fruit
+    fruit_16 = "Pink Fruit in Second Chameleon Fight - R2C12"
+    fruit_17 = "Pink Fruit in Bat Room - R3C3"
+    fruit_18 = "Pink Fruit by Barcode Bunny - R3C5"
+    fruit_19 = "Blue Fruit in Dog Switch Maze - R3C7"  # blue fruit
+    fruit_20 = "Pink Fruit Right of Dog Switch Maze Left - R3C8a"
+    fruit_21 = "Pink Fruit Right of Dog Switch Maze Right - R3C8b"
+    fruit_22 = "Pink Fruit in Dog Elevator Below P. Flame - R3C10"
```

Comment:
```suggestion
    fruit_22 = "Pink Fruit in Dog Elevator By Crow Plinko - R3C10"
```
I think the parallelism with "By P. Flame" on Fruit 14 could benefit from some distinction. Name is workshoppable.

Reply:
```suggestion
    fruit_22 = "Pink Fruit in Dog Elevator by Crow Pachinko - R3C10"
```
"by" should be lowercase, also Pachinko not Plinko

Reply:
"by crow pachinko" would imply this fruit can be gotten by going the crow route, which it cant
```suggestion
    fruit_22 = "Pink Fruit in Dog Elevator near P. Flame Left - R3C10"
```
id rather lump this in with the other fruit there and call them left and right